### DISCUSSION TITLE
Add better UX to maintenance page

### DISCUSSION THREAD

**Discussion by Kukks** [btcpayserver/btcpayserver#2302] [Ideas / Feature Requests]
Currently the maintenance page only lets you "click" and "pray" in that you get no feedback if the command you attempted executed or failed. 

My proposal is to add the feedback system in configurator so that you can get logs in real-time of what the server is doing.  We can also do actions such as show a success if the version changes or the server restarts.

**Comment by pavlenex** [comment]
@Kukks It's still unclear to me how we can improve the maintenance page, can you elaborate on the steps? Do we really want to go the way of embedding feedback system? 

What is the actual problem we're trying to solve?

**Reply by bolatovumar** [reply]
I think what @Kukks meant here is that we can show logs on the same page as the upgrade is being performed so that the user is aware that stuff is happening and it's not just broken.

**Reply by pavlenex** [reply]
Ok, I think that makes sense. Thanks for clarifying @bolatovumar 🥳
